ztp/policygenerator/utils/utils.go

```diff
@@ -1,21 +1,36 @@
 package utils
 
-const ExistOper = "Exists"
-const InOper = "In"
-const DoesNotExistOper = "DoesNotExist"
-const NotInOper = "NotIn"
-const CustomResource = "customResource"
-const SourceCRsPath = "source-crs"
-const FileExt = ".yaml"
-const UnsetStringValue = "__unset_value__"
-const ZtpDeployWaveAnnotation = "ran.openshift.io/ztp-deploy-wave"
-const DefaultCompliantEvaluationInterval = "10m"
-const DefaultNonCompliantEvaluationInterval = "10s"
-const DisableEvaluationInterval = "never"
-
-// ComplianceType of "mustonlyhave" uses significant CPU to enforce. Default to
-// "musthave" so that we realize the CPU reductions unless explicitly told otherwise
-const DefaultComplianceType = "musthave"
+const (
+	ExistOper                             = "Exists"
+	InOper                                = "In"
+	DoesNotExistOper                      = "DoesNotExist"
+	NotInOper                             = "NotIn"
+	CustomResource                        = "customResource"
+	SourceCRsPath                         = "source-crs"
+	FileExt                               = ".yaml"
+	UnsetStringValue                      = "__unset_value__"
+	ZtpDeployWaveAnnotation               = "ran.openshift.io/ztp-deploy-wave"
+	DefaultCompliantEvaluationInterval    = "10m"
+	DefaultNonCompliantEvaluationInterval = "10s"
+	DisableEvaluationInterval             = "never"
+	// DefaultComplianceType ComplianceType of "mustonlyhave" uses significant CPU to enforce. Default to
+	// "musthave" so that we realize the CPU reductions unless explicitly told otherwise
+	DefaultComplianceType = "musthave"
+	// ZtpDeployDeprecationWarningKey annotation key for when source-cr is set to be deprecated. See ZtpDeployDeprecatedSourceCRFileNames for full list
+	ZtpDeployDeprecationWarningKey = "ran.openshift.io/deprecated"
+	// ZtpDeployDeprecationWarningValue annotation value for when source-cr is set to be deprecated. See ZtpDeployDeprecatedSourceCRFileNames for full list
+	ZtpDeployDeprecationWarningValue = "true"
```

Comment:
Would it make sense to have a longer message here to indicate why/when it is deprecated. ie, some sort of informational text:
`ran.openshift.io/deprecation-warning: "Source CR StorageCatSource is deprecated. Use DefaultCatSource instead"`